lib/screens/home/widgets/drawer/list_tile.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+class MListTileWidget extends StatelessWidget {
```

Comment:
I need to distinguish my Widget name from the library provided Widget.  I have removed the tailing(widget), but I have kept the M (as in MListTile).

Reply:
How about naming the widget as something like 'ListItem' or 'MenuItem'? That way the name of the widget would be descriptive and at the same time it would be distinctive from the standard ListTile widget.